### ISSUE TITLE
Can't run ungit, async path unexpected identifier

### ISSUE THREAD

**Issue by NathanOrme** [FredrikNoren/ungit#1204]
Running on node v6.16.0,npm v3. 10.10, ungit version 1.4.44.

Get error on line 11 of index.js: const wslToWindowsPath = async path... SyntaxError: Unexpected Identifier on path. 
![15597326900156399218011382706702](https://user-images.githubusercontent.com/18661760/58951890-33743080-878a-11e9-82f7-cb0a08146867.jpg)

**Comment by ZimGil** [comment]
Is this on Linux?

**Comment by NathanOrme** [comment]
It is @zimgil

**Comment by ZimGil** [comment]
Than it's a dup of #1203 
I will comment there...

**Comment by ylecuyer** [comment]
IMO readme should be updated to say ungit support node >=8 as we can see in the travis config file. WDYT @jung-kim  ?

**Comment by tommai78101** [comment]
Also have this issue on CentOS 7. It's very likely this issue has an impact on RHEL flavored Linux OSes. Will also comment on #1203.

**Comment by jung-kim** [comment]
Yup, can you share your node version you are running?  
It probably is at the version that is red for the [async func](https://node.green/#ES2017-features-async-functions) so once you upgrade your node you should be fine.

@tommai78101 This particular error is independent from the OS and solely contingent on the node version.  Please open a ticket if you see different errors on CentOS, or doesn't get fixed after node upgrade.

@ylecuyer yes I will update the README.

**Comment by tommai78101** [comment]
@jung-kim I am seeing the same exact error and stack trace as what was taken in the screenshot (or camera capture). Because of this, I don't see the need to open a new ticket.

**Comment by jung-kim** [comment]
README is updated with 8.16.0 as that is the minimal version of node we are running travis tests with.

And yeah once you upgrade node version >= 8.16.0 you should be fine
